kubernetes/openshift/blueprint.yaml

```diff
@@ -0,0 +1,208 @@
+        namespace: { get_input: namespace }
+    relationships:
+    - type: cloudify.kubernetes.relationships.managed_by_master
+      target: kubernetes_master
+    - type: cloudify.relationships.connected_to
+      target: app_svc
+
+  app_svc:
+    type: cloudify.kubernetes.resources.Service
+    properties:
+      definition:
+        apiVersion: v1
+        kind: Service
+        metadata:
+          name: { concat: [{ get_property: [app, definition, metadata, name] }, '-svc'] }
+          labels:
+            app: { get_input: app_name }
+            manager: cloudify
+            tier: app
+        spec:
+          selector:
+            app: { get_input: app_name }
+            manager: cloudify
+            tier: app
+          ports:
+          - port: 80
+            targetPort: { get_input: container_port }
+      options:
+        namespace: { get_input: namespace }
+      validate_resource_status: false
+    relationships:
+    - type: cloudify.kubernetes.relationships.managed_by_master
+      target: kubernetes_master
+    - type: cloudify.relationships.contained_in
+      target: app
+
+  app:
+    type: cloudify.kubernetes.resources.Deployment
+    properties:
+      definition:
+        apiVersion: apps/v1
+        kind: Deployment
+        metadata:
+          name: { concat: [{ get_input: app_name }, '-app'] }
+          # # If you want to have OpenShift show the arrows connecting services/deployments, 
+          # # you should use the connects-to annotation - example below. 
+          # annotations:
+          #   app.openshift.io/connects-to: { get_property: [my_database, definition, metadata, name] }
+          labels:
+            app: { get_input: app_name }
+            manager: cloudify
+            tier: app
+        spec:
+          selector:
+            matchLabels:
+              app: { get_input: app_name }
+              manager: cloudify
+              tier: app
+          replicas: 1
+          template:
+            metadata:
+              labels:
+                app: { get_input: app_name }
+                manager: cloudify
+                tier: app
+            spec:
+              containers:
+              - name: { get_property: [SELF, definition, metadata, name] }
+                image: { get_input: container }
+                ports:
+                - containerPort: { get_input: container_port }
+                  name: http-server
+                resources:
+                  limits:
+                    cpu: 1
+                    memory: 128Mi
+                  requests:
+                    cpu: 1
+                    memory: 128Mi
+              #   # Example volume mount from PVC
+              #   volumeMounts:
+              #   - mountPath: "/var/www"
+              #     name: "pvol"
+              # volumes:
+              # - name: "pvol"
+              #   persistentVolumeClaim:
+              #     claimName: { get_property: [my_volume_claim, definition, metadata, name] }
```

Comment:
These are commented out because some case doesn't use them? Can you explain in comments under what circumstances the comments should be removed or just remove them if it's not important?